consensus/hotstuff/votecollector/combined_vote_processor.go

```diff
@@ -19,7 +19,13 @@ func CombinedVoteProcessorFactory(log zerolog.Logger, proposal *model.Proposal)
 	processor := &CombinedVoteProcessor{
 		log:   log,
 		block: proposal.Block,
-		done:  *atomic.NewBool(false),
+		// TODO: initialize the following dependencies
+		// stakingSigAggtor
+		// rbSigAggtor
+		// rbRector
+		// onQCCreated
+		// packer
```

Comment:
Test cases can not be implemented until we figure out how to pass in these dependencies.